.github/workflows/all-tests.yml

```diff
@@ -23,7 +23,11 @@ jobs:
       max-parallel: 4
       matrix:
         operating-system: [ubuntu-latest, macos-latest]
-        version: [14.x, 16.x, 18.x, 19.x]
+        version: [16.x, 18.x, 20.x, 21.x]
```

Comment:
Node 16.x is technically unsupported, but...
Node 18.x, 20.x and 21.x are the current, supported version.